native/dotnet-sdk-6.0/Makefile

```diff
@@ -1,7 +1,7 @@
 PKG_NAME = dotnet-sdk-6.0
 # Version 6.0.9, SDK 6.0.401
```

Comment:
this should be updated to:

# Version 6.0.13, SDK 6.0.405

Reply:
done